fiona/_env.pyx

```diff
@@ -186,6 +186,59 @@ cdef class ConfigEnv(object):
         return {k: get_gdal_config(k) for k in self.options}
 
 
+class GDALDataFinder(object):
+    """Finds GDAL and PROJ data files"""
+
+    def search(self, prefix=None):
+        """Returns GDAL_DATA location"""
+        path = self.search_wheel(prefix or __file__)
+        if not path:
+            path = self.search_prefix(prefix or sys.prefix)
+            if not path:
+                path = self.search_debian(prefix or sys.prefix)
+        return path
```

Comment:
I think I prefer early returns instead of nested ifs, but it's a personal preference.
```python
    def search(self, prefix=None):
        """Returns GDAL_DATA location"""
        path = self.search_wheel(prefix or __file__)
        if path:
            return path
        path = self.search_prefix(prefix or sys.prefix)
        if path:
            return path
        path = self.search_debian(prefix or sys.prefix)
        if path:
            return path
```

Reply:
Yeah, I could go either way. Multiple returns isn't bad when they're so close together.